3rdparty/pcrs/pcrs-0.0.3/pcrs.3.in

```diff
@@ -0,0 +1,488 @@
+.\" Copyright (c) 2001-2003 Andreas S. Oesterhelt <oes@oesterhelt.org>
+.\"
+.\" This is free documentation; you can redistribute it and/or
+.\" modify it under the terms of the GNU General Public License as
+.\" published by the Free Software Foundation; either version 2 of
```

Comment:
Indeed, linking to other GPL code is going to be problematic, the license won't allow us to release our Enterprise product.